README.md

```diff
@@ -174,4 +174,4 @@ We currently have plans to support:
 
 This project is under an MIT license
 
-Powered by: [MagenTys](http://magentys.io)
+Powered by: [MagenTys](http://magentys.io) [MechanicalRock](https://www.mechanicalrock.io/#/)

```

Comment:
should probably remove the #/ - so https://www.mechanicalrock.io/

Reply:
[Mechanical Rock](https://mechanicalrock.io)

Reply:
Done